topics/statistics/tutorials/CNN/tutorial.md

```diff
@@ -0,0 +1,426 @@
+---
+layout: tutorial_hands_on
+
+title: Introduction to convolutional neural networks (CNN)
+zenodo_link:
+questions:
+- What is a convolutional neural network (CNN)?
+- What are some applications of CNN?
+objectives:
+- Understand the inspiration behind CNN and learn the CNN architecture
+- Learn the convolution operation and its parameters
+- Learn how to create a CNN using Galaxy's deep learning tools
+- Solve an image classification problem on MNIST digit classification dataset using CNN in Galaxy
+requirements:
+  -
+    type: internal
+    topic_name: statistics
+    tutorials:
+      - intro_deep_learning
+time_estimation: 2H
+contributors:
+- kxk302
+
+---
+
+
+
+
+# Introduction
+{:.no_toc}
+
+<!-- This is a comment. -->
```

Comment:
```suggestion
```

On the tutorial formatting side I don't think I have any comments currently, this looks really well formatted from the start! Thanks for preparing it so well.

Reply:
Thx @hexylena! Applied the change.